libraries/theme/src/main/kotlin/io/element/android/libraries/theme/ElementTheme.kt

```diff
@@ -62,19 +65,25 @@ object ElementTheme {
 
     /**
      * The current Material 3 [ColorScheme] provided by [ElementTheme], coming from [MaterialTheme].
+     * In Figma, these colors usually have the `M3/` prefix.
      */
     val materialColors: ColorScheme
         @Composable
         @ReadOnlyComposable
         get() = MaterialTheme.colorScheme
 
     /**
-     * Material 3 [Typography] tokens.
+     * Material 3 [Typography] tokens. In Figma, these have the `M3/` prefix.
      */
-    val typography: Typography
+    val materialTypography: Typography
         @Composable
         @ReadOnlyComposable
         get() = MaterialTheme.typography
+
+    /**
+     * Compound [Typography] tokens. In Figma, these have the `Android/font/` prefix.
+     */
+    val typography: TypographyTokens = TypographyTokens
```

Comment:
Can we adopt a more consistent naming scheme?

e.g.

`materialColors` + `materialTypography` (for material stuff)
`compoundColors` + `compoundTypography ` (for compound stuff)
`colors` (for legacy stuff)

OR

`materialColors` + `materialTypography` (for material stuff)
`colors` + `typography ` (for compound stuff)
`elementColors` (for legacy stuff)

The second example does away with prefixes for the compound stuff so that is naturally the "default" choice.

Reply:
Yeah, the 2nd option sounds nice. I thought about using `legacyColors` for those but I wasn't sure it was accurate as some of them are not legacy as in coming from the old app, but they are legacy as in we shouldn't be using it anymore.

Reply:
Can we just remove what should not be used anymore?

Reply:
We shouldn't use them anymore but they might still be in the designs and we'll need to use them if there's no replacement, that's why I'm keeping them here. Also, by having all of them referenced here it's easier to just search for usage of this field if we wanted to do a batch replace in the future.

Reply:
Some new changes in https://github.com/vector-im/element-x-android/pull/700/commits/076c282d751e52ee02b7406389a16256061155e3 to rename the helper properties and enforce their usage.